src/runtime/nitro/plugins/04-cspSsgHashes.ts

```diff
@@ -102,8 +102,10 @@ export default defineNitroPlugin((nitroApp) => {
     // Generate CSP rules
     const csp = security.headers.contentSecurityPolicy
     const headerValue = generateCspRules(csp, scriptHashes, styleHashes)
-    // Insert CSP in the http meta tag
-    html.head.push(`<meta http-equiv="Content-Security-Policy" content="${headerValue}">`)
+    // Insert CSP in the http meta tag if enabled
+    if (enabled) {
```

Comment:
suggestion: I wonder if we shouldn't instead have this check at the `module.ts` level [here](https://github.com/Baroshem/nuxt-security/blob/a6b9c02150ca128240bdfb3af7474988f8b495fc/src/module.ts#L306). 

This way we would be able to not trigger any ssg plugins at all if we are not interested in having them.

Maybe, if you only need to not add meta tag while remain having ssg hashes and scripts, the better option would be to add a `meta` config option instead of `enabled`, WDYT?

Because my understanding of `enabled` boolean flag in `ssg` object would be to disable the work of SSG completely by this module

Reply:
Completely agree!